src/pages/settings/index.js

```diff
@@ -36,6 +36,9 @@ store
       };
     }
 
+    // Sync options with background
+    chrome.runtime.sendMessage({ action: 'syncOptions' });
```

Comment:
should we await here? we don't want UI to flicker in case option change

Reply:
We already discussed awaiting the sync process in another context, and our decision was to not wait for it. It can be long or failing for many reasons (bad internet connection etc) - The settings page are available immediately for the user (they are local).